src/pybind/mgr/dashboard/run-frontend-e2e-tests.sh

```diff
@@ -2,29 +2,40 @@
 
 set -e
 
+$CLUSTERS=("c1","c2")
+
 start_ceph() {
     cd $FULL_PATH_BUILD_DIR
 
-    MGR=2 RGW=1 ../src/vstart.sh -n -d
-    sleep 10
+    for cluster in ${CLUSTERS[@]}; do
+        MGR=2 RGW=1 ../src/mstart.sh $cluster -n -d
+        sleep 10
+    done
 
     set -x
+    alias ceph='../src/mrun c1 ceph'
+    alias ceph2='../src/mrun c2 ceph'
 
     # Create an Object Gateway User
-    ./bin/ceph dashboard set-rgw-credentials
+    ceph dashboard set-rgw-credentials
+    ceph2 dashboard set-rgw-credentials
 
     # Set SSL verify to False
-    ./bin/ceph dashboard set-rgw-api-ssl-verify False
+    ceph dashboard set-rgw-api-ssl-verify False
+    ceph2 dashboard set-rgw-api-ssl-verify False
 
-    CYPRESS_BASE_URL=$(./bin/ceph mgr services | jq -r .dashboard)
+    CYPRESS_BASE_URL=$(ceph mgr services | jq -r .dashboard)
+    CYPRESS_BASE_URL_CEPH2=$(ceph2 mgr services | jq -r .dashboard)
```

Comment:
I'd refrain from using the `CYPRESS_BASE_...` name, since it denotes that it's Cypress the consumer of this variable (and it's actually us).

Reply:
Make sense. Renamed it.